components/match2/commons/match_summary_base.lua

```diff
@@ -184,6 +185,21 @@ function Footer:addLinks(links)
 		end
 	end
 
+	local processedLinks = {}
+	Array.forEach(MATCH_LINK_PRIORITY, function(linkTypePrefix)
+		for linkType, link in Table.iter.pairsByPrefix(links, linkTypePrefix, {requireIndex = false}) do
+			processLink(linkType, link)
+			processedLinks[linkType] = true
+		end
+	end)
+
+	for linkType, link in Table.iter.spairs(links) do
+	    -- Handle linktypes not already processed via priority list
+		if not processedLinks[linkType] then
+			processLink(linkType, link)
+		end
+	end
```

Comment:
```suggestion
	Array.forEach(MATCH_LINK_PRIORITY, function(linkType)
		for linkKey, link in Table.iter.pairsByPrefix(links, linkType, {requireIndex = false}) do
			processLink(linkKey, link)
			processedLinks[linkKey] = true
		end
	end)

	for linkKey, link in Table.iter.spairs(links) do
	    -- Handle links not already processed via priority list
		if not processedLinks[linkKey] then
			processLink(linkKey, link)
		end
	end
```

Renamed two vars (linkTypePrefix -> linkType and linkType -> linkKey), I think these names makes more sense? Feel free to disagree